doc/api/tls.md

```diff
@@ -688,7 +694,17 @@ For RSA keys, the following properties may be defined:
    `'B56CE45CB7...'`.
 * `pubkey` {Buffer} The RSA public key.
 
-
+For EC keys, the following properties may be defined:
+* `ecBits` {number} The bit-strength of the elliptic curve. Example: `256`.
+* `ecPubKey` {Buffer} The EC public key.
```

Comment:
It's a minor inconsistency to camel-case `PubKey` when it's spelled `pubkey` for RSA keys. Perhaps `ecPubkey`?

Reply:
How about just `pubkey`? Then its the same as RSA. The cert property names are so wildly varying, from `infoAccess` to `ext_key_usage` to `subjectaltname` I figured rather than try any attempt at consistency, I'd just use our modern naming convention. I could do `ecPubkey`, but it feels a bit like the worst of both worlds. It isn't the same as RSA, but isn't camelCase either, it has the same RSA capitalization error.

Reply:
`pubkey` WFM but makes it inconsistent with the other `ec*` properties. You could strip the ec prefix, I don't think that'd conflict with other properties, but makes it less clear they're EC-only properties. I'll just leave it up to you.

Reply:
I stripped ec, the "curve" is a pretty clear indication that its an elliptic curve. And I added `bits` for RSA for symmetricality. PTAL.